doc/lua_api.md

```diff
@@ -7921,6 +7921,26 @@ child will follow movement and rotation of that bone.
 * `get_bone_overrides()`: returns all bone overrides as table `{[bonename] = override, ...}`
 * `set_properties(object property table)`
 * `get_properties()`: returns a table of all object properties
+* `set_observers(observers)`: sets observers (players this object is sent to)
+    * If `observers` is `nil`, the object's observers are "unmanaged":
+      The object is sent to all players as governed by server settings. This is the default.
+    * `observers` is a "set" of player names: `{[player name] = true, [other player name] = true, ...}`
+        * Since this is a set, the values need to be `true`.
```

Comment:
I'm a fan of consistency, but I think the term "flag field" would be confusing here, especially since we support some additional specifier formats there (strings). I also find the term less intuitive.

I can properly define the term. To me "set" is sort of "extended" Lua lingo, just like you'd probably call a table with keys 1, 2, ..., n a "list". But it seems this notion is not that widespread so I should probably clarify it.